controls/nist_rhcos4.yml

```diff
@@ -6784,8 +6784,12 @@ controls:
   - high
   - moderate
 - id: IA-2(4)
-  status: pending
+  status: not applicable
   notes: |-
+    This is out of scope, RHCOS does not provide MFA natively, only through 3rd party IDM such as LDAP. 
+    CoreOS Tectonic User Auth
+    https://cloud.redhat.com/learn/topics/tectonic
```

Comment:
This page says that tectonic is EOL. I wouldn't mention it. Let's mention LDAP, AD or RHIDM instead.
But a bigger question is, should we just punt on the user management in RHCOS altogether like we did for STIG and even make sure that e.g. SSSD is disabled and the only way to access the nodes is through `oc debug`?
Because..if we do suggest that you can use 2FA to log in to RHCOS, what are our options other than letting someone install SSSD or hand-configuring SSH with certificates/yubikeys/whatnot?

Also, remove the blob before about control response is planned.

Reply:
Thanks for checking out the PR, I think that makes sense, maybe we should say, the only way to access the nodes is through `oc debug` because SSSD is disabled.

Reply:
So all the authentication is handled on OCP, through a third party IDP

Reply:
That's really a design question of sorts. I don;t think that moderate had this kind of requirement for direct access to nodes. I think we should consider what our options are and also take into consideration that after the default installation, there is the `core` account which logs in with an ssh key...is it enough to make sure that the ssh key is protected by a passphrase? Or do we need to even disable ssh?

Let's gather all the answers and not rush the PR in..

Reply:
That makes sense, let's hold this PR for now, yes there is a core account with ssh key, I think maybe we can also block the ssh port on the network firewall.

Reply:
/hold